app/src/main/java/com/cornellappdev/score/screen/HomeScreen.kt

```diff
@@ -0,0 +1,71 @@
+package com.cornellappdev.score.screen
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.Alignment
+import com.cornellappdev.android.score.data.models.GameCardData
+import com.cornellappdev.score.components.SportCard
+import com.cornellappdev.score.components.UpcomingGameCard
+import com.cornellappdev.score.components.UpcomingGamesCarousel
+import com.cornellappdev.score.theme.poppinsFamily
+import com.cornellappdev.score.util.PENN_GAME
+import com.cornellappdev.score.util.PRINCETON_GAME
+import com.cornellappdev.score.util.gameList
+import com.example.score.R
+import com.google.accompanist.pager.ExperimentalPagerApi
+
+@Composable
+fun HomeScreen() {
+    Column( verticalArrangement = Arrangement.spacedBy(16.dp, Alignment.Top))
+    {
+        UpcomingGamesCarousel(gameList)
+        Column(modifier = Modifier.padding(horizontal = 24.dp)) {
+            Text(text = "Game Schedule", style = TextStyle(
+                fontSize = 24.sp,
+                fontFamily = poppinsFamily,
+                fontWeight = FontWeight(600),
+                color = Color(0xFF333333),
+                )
+            )
```

Comment:
Hmmm, the style here seems like something we can generalize so that you don't have to keep writing it. Is this something that follows a text style (i.e. `title1`, `header1`, etc.) in the Figma? If so, add that to the `Style` singleton you made before and use that style here.